aries-site/src/components/accessibility/Video.js

```diff
@@ -15,9 +15,9 @@ export const AccessVideo = () => {
 
   return (
     <Box width="large" pad={{ bottom: 'large' }}>
-      <video 
```

Comment:
were these changes showing up on your eslint?

Reply:
Yes they were -- we can connect with Amanda to make sure her eslint env is set up correctly for future PRs.

Reply:
sounds like a plan